Appraisals

```diff
@@ -19,3 +19,9 @@ appraise "activerecord-7.1" do
   gem "activesupport", "~> 7.1"
   gem "standard", "~> 1.16"
 end
+
+appraise "activerecord-7.2" do
+  gem "activerecord", "~> 7.2"
+  gem "activesupport", "~> 7.2"
+  gem "standard", "~> 1.16"
+end
```

Comment:
Could you please add a newline here? It breaks Standard linter.

Reply:
Actually, nevermind, I am gonna merge as is because I still wanna do some other changes, gonna change this as well.